subtests/docker_cli/build/build.py

```diff
@@ -225,7 +225,8 @@ def basic_postprocess(build_def, command, parameter):
             if OutputGood(build_def.dockercmd.cmdresult,
                           ignore_error=True,
                           skip=['error_check']):
-                return build_def.dockercmd.cmdresult.exit_status != 0
+                # FIXME: this is likely to fail in docker-1.10; bz1097344
+                return build_def.dockercmd.cmdresult.exit_status == 1
```

Comment:
Nice, this one actually is helpful, even if it winds up being permanently-temporary (like the rest) :grinning: